### ISSUE TITLE
Let's use Respect\Validator

### ISSUE THREAD

**Issue by daison12006013** [progsmile/request-validator#17]
Hi @progsmile,

We could probably change our rules to use the respect validator class.

We could still achieve the other rules still, by I might gonna refactor the validator class you have and let us see.

I'm also planning to use this package though inside [slayer's framework](http://github.com/phalconslayer/slayer)

I might gonna initially request a PR within this week.

**Comment by sergeyklay** [comment]
:+1:

**Comment by progsmile** [comment]
Hi @daison12006013, 
You may change them, but would it be compatible with current validator api? - If not, could you create the second version of package (2.0.0) ?)

**Comment by daison12006013** [comment]
Hi @progsmile,

It will be compatible, try to check my forked repo http://github.com/daison12006013/request-validator/tree/dev

I modified some of the rules already.

Upon checking, the Validator class seems to be untouched, I added a function respect(...) on BaseRule, which works well.

I can't see any releases on this package, let's try to release a 1.0.0 tag later on.

Have you added this repo on packagist.org?
